driver/csiplugin/connectors/connectors.go

```diff
@@ -42,6 +42,7 @@ type SpectrumScaleConnector interface {
 	UnlinkFileset(filesystemName string, filesetName string) error
 	//ListFilesets(filesystemName string) ([]resources.Volume, error)
 	ListFileset(filesystemName string, filesetName string) (Fileset_v2, error)
+	GetFilesetsParentId(filesystemName string, parentId int) ([]Fileset_v2, error)
```

Comment:
should the filter be based on inodespace, because parentID does not work in case of unlinked fileset

Reply:
Done, using inodeSpace instead of parentID.